README.md

```diff
@@ -56,6 +56,18 @@ It has specific TDR commands around realm import, and is copied from the default
 
 The standalone-ha.xml is mostly the standard configuration for keycloak with a few changes to get it to work with the load balancer. Some of these are discussed in the keycloak [documentation](https://www.keycloak.org/docs/latest/server_installation/#_setting-up-a-load-balancer-or-proxy)
 
+## TDR Theme
+
+TDR has its own Keycloak theme: `tdr`
+
+The TDR theme makes use of the standard Keycloak base theme, with specific overrides to the theme resources where required.
+
+The TDR theme styling is a combination of [Gov.UK Design System](https://github.com/UKHomeOffice/keycloak-theme-govuk), with TDR specific overrides. 
```

Comment:
Should this link go to https://design-system.service.gov.uk/?

Reply:
Grr, yes. I blame cut and paste.